.github/actions/maven-build-and-tag/action.yml

```diff
@@ -117,6 +131,16 @@ runs:
       run: |
         GITHUB_PR_NUMBER=PR-${{ github.event.pull_request.number }}
         echo "preview-version=0.0.1-$GITHUB_PR_NUMBER-SNAPSHOT" >> $GITHUB_OUTPUT
+        echo "GITHUB_PR_NUMBER=$GITHUB_PR_NUMBER" >> $GITHUB_ENV
```

Comment:
Nitpick: maybe keep it in GITHUB_OUTPUT instead of pushing to the shared env

Reply:
It is OK to use Github env for it